examples/plugin-compat/rspack.config.js

```diff
@@ -20,14 +19,14 @@ const config = {
 		filename: "[contenthash:8].js"
 	},
 	optimization: {
-		minimize: true,
-		minimizer: [
-			new minifyPlugin({
-				minifier: "terser",
-				target: "es6",
-				css: true
-			})
-		]
+		minimize: false, // Disabling minification because it takes too long on CI
+		// minimizer: [
+			// new minifyPlugin({
+				// minifier: "terser",
+				// target: "es6",
+				// css: true
+			// })
+		// ]
```

Comment:
examples/plugin-compat build: Rspack 0.3.11 compiled with 2 warnings in 37.29 s

Reply:
@bvanjoi let's move this out to https://github.com/web-infra-dev/rspack-compat

Reply:
Did you intend to refer to `﻿@rspack/plugin-minify`?

Reply:
or other plugins such as `fork-ts-checker-webpack-plugin`/`license-webpack-plugin`/`generate-package-json-webpack-plugin`?

Reply:
done: https://github.com/web-infra-dev/rspack-compat/pull/2

Reply:
@Boshen all of the loaders and plugins should be moved to rspack-compat and trigger rspack-compat by ecosystem-ci

Reply:
let's merge this first